config/docker/cros-baseline/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM debian:bullseye-slim
+MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    apt-transport-https \
+    ca-certificates \
+    ssh
+
+RUN apt-get update && apt-get install -y \
+    inetutils-ping
```

Comment:
Add git:
```
Step 11/17 : RUN git clone     https://chromium.googlesource.com/chromiumos/chromite     $HOME/trunk/chromite
 ---> Running in 31f0ad9fb74d
/bin/sh: 1: git: not found
The command '/bin/sh -c git clone     https://chromium.googlesource.com/chromiumos/chromite     $HOME/trunk/chromite' returned a non-zero code: 127
```

Reply:
Fixed that, sorry forgot to run manual build